front/lib/api/tables.ts

```diff
@@ -100,14 +99,9 @@ export async function deleteTable({
       message: "Failed to delete table.",
     });
   }
-  await AgentTablesQueryConfigurationTable.destroy({
-    where: {
-      dataSourceWorkspaceId: owner.sId,
-      // TODO(DATASOURCE_SID); state storing the datasource name
-      dataSourceId: dataSource.name,
-      tableId,
-    },
-  });
+  // We do not delete the related AgentTablesQueryConfigurationTable entry if any.
+  // This is because the table might be created again with the same name and we want to keep the configuration.
```

Comment:
identation looks a bit off?

Reply:
LGTM I kept it right before returning the OK!